source/agora/node/BlockStorage.d

```diff
@@ -742,9 +742,9 @@ public class BlockStorage : IBlockStorage
             File idx_file = File(this.index_path, "a+b");
             idx_file.seek(0, SEEK_END);
 
-            serializePart(height, (scope v) => idx_file.rawWrite(v));
+            serializePart(height, (scope v) => idx_file.rawWrite(v), CompactMode.No);
```

Comment:
Where are the changes for the deserialize part? It's serializing in non-compact mode, but it's deserializing in compact mode. Won't this cause issues?

Reply:
This part was discussed with @Geod24  to proceed with a separate PR.

Reply:
What was said in the discussion? Was it on github? I'm not aware of it, sorry.

Reply:
Ok Mathias explained it to me.